src/components/DNSenumTool/DNSenumTool.tsx

```diff
@@ -223,6 +243,16 @@ const DnsenumTool = () => {
                     <Button type={"submit"}>Start Enumeration</Button>
                     {SaveOutputToTextFile_v2(output, allowSave, hasSaved, handleSaveComplete)}
                     <ConsoleWrapper output={output} clearOutputCallback={clearOutput} />
+
+                    <Modal opened={modalOpened} onClose={() => setModalOpened(false)} title="Missing Commands">
```

Comment:
Interesting implementation but I recommend that you add the conventional component installationModal. Here is an example of what it looks like in WhatWeb: 

![image](https://github.com/user-attachments/assets/3f7c0372-d261-459f-b4d6-1680fe62579a)
```suggestion
                 
```